### ISSUE TITLE
bot need update

### ISSUE THREAD

**Issue by zamiguinbra** [dante4rt/getgrass-bot#38]
the bot needs an update to remove dead proxy to have better activate connection with points
![grass](https://github.com/user-attachments/assets/a1070677-134f-43c5-9ba9-09772dce7b28)

**Comment by corebroker** [comment]
x2. It is not connecting all proxies

**Comment by corebroker** [comment]
> the bot needs an update to remove dead proxy to have better activate connection with points

Did you manage to fix?

**Comment by zamiguinbra** [comment]
> > the bot needs an update to remove dead proxy to have better activate connection with points
> 
> Did you manage to fix?

No, I sent an email to the creator and he didn't respond.   There are other bots that are working in Python, all the proxies are active without error, unfortunately I couldn't work, just this one from Dante

**Comment by corebroker** [comment]
> > > the bot needs an update to remove dead proxy to have better activate connection with points
> > 
> > 
> > Did you manage to fix?
> 
> No, I sent an email to the creator and he didn't respond. There are other bots that are working in Python, all the proxies are active without error, unfortunately I couldn't work, just this one from Dante

Which one working for ya?

**Comment by zamiguinbra** [comment]
> > > > the bot needs an update to remove dead proxy to have better activate connection with points
> > > 
> > > 
> > > Did you manage to fix?
> > 
> > 
> > No, I sent an email to the creator and he didn't respond. There are other bots that are working in Python, all the proxies are active without error, unfortunately I couldn't work, just this one from Dante
> 
> Which one working for ya?

This bot here is working, but it only connects a maximum of 30 proxies that are scored, the rest are at zero points

**Comment by zamiguinbra** [comment]
> > > > the bot needs an update to remove dead proxy to have better activate connection with points
> > > 
> > > 
> > > Did you manage to fix?
> > 
> > 
> > No, I sent an email to the creator and he didn't respond. There are other bots that are working in Python, all the proxies are active without error, unfortunately I couldn't work, just this one from Dante
> 
> Which one working for ya?

There is another bot that can connect to 3000 proxies, that's 10 million points per day, it worked for some people
